tools/update-eslint-all.js

```diff
@@ -23,9 +24,19 @@ for (const [ruleId, rule] of builtInRules) {
     }
 }
 
-//------------------------------------------------------------------------------
-// Public Interface
-//------------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+// Main
+//-----------------------------------------------------------------------------
+
+const code = `/*
+ * WARNING: This file is autogenerated using the tools/update-eslint-all.js
+ * script. Do not edit manually.
+ */
+"use strict";
+
+/* eslint quote-props: off -- autogenerated so don't lint */
+
+module.exports = Object.freeze(${JSON.stringify({ rules: allRules }, null, 4)});
+`;
 
-/** @type {import("../lib/shared/types").ConfigData} */
-module.exports = { rules: allRules };
+fs.writeFileSync("./packages/js/src/eslint-all.js", code, "utf8");
```

Comment:
```suggestion
fs.writeFileSync("./packages/js/src/configs/eslint-all.js", code, "utf8");
```

Reply:
Oops, thanks! Missed during the refactor.